### ISSUE TITLE
[Bug]: execute revoke moadmin xxx error

### ISSUE THREAD

**Issue by heni02** [matrixorigin/matrixone#5164]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior
revoke moadmin from root;
![image](https://user-images.githubusercontent.com/113406637/191490635-8fc8e54c-d04d-4e51-8886-1705d073e03e.png)

**Comment by daviszhen** [comment]
it was fixed at #5411

**Comment by heni02** [comment]
confirm，closed
